src/utilities/filetypes/EpwFile.cpp

```diff
@@ -1641,7 +1641,7 @@ namespace openstudio{
 
   Date EpwDataPoint::date() const
   {
-    return Date(MonthOfYear(m_month), m_day); // , m_year);
+    return Date(MonthOfYear(m_month), m_day, m_year);
```

Comment:
`EpwDataPoint::date()` now returns whatever year was actually in the line (datapoint) of the epw file